nixos/modules/programs/coolercontrol.nix

```
@@ -13,18 +13,15 @@ in
   options = {
     programs.coolercontrol = {
       enable = lib.mkEnableOption "CoolerControl GUI & its background services";
-
-      nvidiaSupport = lib.mkOption {
-        type = lib.types.bool;
-        default = lib.elem "nvidia" config.services.xserver.videoDrivers;
-        defaultText = lib.literalExpression "lib.elem \"nvidia\" config.services.xserver.videoDrivers";
-        description = ''
-          Enable support for Nvidia GPUs.
-        '';
-      };
     };
   };
 
+  imports = [
+    (lib.mkRemovedOptionModule [ "programs" "coolercontrol" "nvidiaSupport" ] ''
```

Just adding a quick note here so we know when this option was added and thus when it's fine to drop it completely.

(I don't know if this is really an established practice for module stuff, but it is with package deprecations/removals - see i.e. the changes to `pkgs/top-level/aliases.nix` in https://github.com/NixOS/nixpkgs/commit/a780e166886ebae6b6adfbe124244282e54d2fc8.)

```suggestion
    # Added 2025-10-25
    (lib.mkRemovedOptionModule [ "programs" "coolercontrol" "nvidiaSupport" ] ''
```